prow/pjutil/pjutil_test.go

```diff
@@ -1211,6 +1213,87 @@ func TestSpecFromJobBase(t *testing.T) {
 				return nil
 			},
 		},
+		{
+			name: "Verify DecorateExtraRefs default true",
+			jobBase: config.JobBase{
+				UtilityConfig: config.UtilityConfig{
+					DecorationConfig: &prowapi.DecorationConfig{
+						BloblessFetch: &trueVar,
```

Comment:
```suggestion
						BloblessFetch: ptr.To(true),
```

nit: that utility makes it a little easier - the reader no longer has to trust that `trueVar` actually holds `true` ;)

Reply:
Good point. Rather than import a package not used by Prow, I noticed that pjutil.go already has boolPtr(). Will use that.

Reply:
Feel free not to change this - only a nit.

Reply:
Done